redis/src/connection.rs

```diff
@@ -64,6 +64,10 @@ pub enum ConnectionAddr {
         host: String,
```

Comment:
add to doc "This name should remain the DNS name, for the purpose of TLS hostname verification. IP addresses can be saved in `socket_addr`."